backend/src/org/akvo/lumen/lib/dashboard.clj

```diff
@@ -0,0 +1,29 @@
+(ns org.akvo.lumen.lib.dashboard
+  (:require [org.akvo.lumen.lib.dashboard-impl :as impl]))
+
+
+(defn all
+  "Returns all dashboards."
+  [tenant-conn]
+  (impl/all-dashboards tenant-conn))
+
+(defn create
+  "Creates a new dashboard. Body should include keys and spec conform
```

Comment:
@kardan This docstring is still wip 👍 

I still wanted to mention that `"Body"` perhaps does not make sense in this context. Perhaps better (when we have clojure.spec/schema) to point to the var holding the spec definition. Something like `spec must conform with org.akvo.lumen.lib.dashboard/spec`

When I see `body` in a docstring I always think of macros, where body would be one or more expressions